--- partclone-0.2.48/configure.ac.wiget	2013-02-10 21:59:03.925355329 +0100
+++ partclone-0.2.48/configure.ac	2013-02-10 22:07:35.071790552 +0100
@@ -1,13 +1,14 @@
 AC_INIT([Partclone], [0.2.48], [thomas@nchc.org.tw])
 #AC_CONFIG_AUX_DIR([build-aux])
 #AC CONFIG SRCDIR([src/main.c])
-AM_INIT_AUTOMAKE([-Wall -Werror foreign])
+AM_INIT_AUTOMAKE([-Wall foreign])
 AM_GNU_GETTEXT_VERSION([0.16.1])
 AM_GNU_GETTEXT([external])
 AC_PROG_CC
 AC_PROG_INSTALL
 AC_PATH_PROG(RM, rm, rm)
 AC_PROG_LN_S
+AM_PROG_CC_C_O
 
 # Enable large file support.
 AC_SYS_LARGEFILE
--- partclone-0.2.48/src/Makefile.am.wiget	2013-02-10 22:02:38.249268004 +0100
+++ partclone-0.2.48/src/Makefile.am	2013-02-10 22:06:41.313749533 +0100
@@ -5,8 +5,11 @@ TOOLBOX = srcdir=$(top_srcdir) builddir=
 
 BTRFS_SOURCE= btrfs/crc32c.c btrfs/disk-io.c btrfs/extent-tree.c btrfs/inode-map.c btrfs/rbtree.c btrfs/volumes.c btrfs/ctree.c btrfs/extent-cache.c btrfs/file-item.c btrfs/print-tree.c btrfs/root-tree.c btrfs/dir-item.c btrfs/extent_io.c btrfs/inode-item.c btrfs/radix-tree.c btrfs/utils.c btrfs/crc32c.h btrfs/extent-cache.h btrfs/ioctl.h btrfs/print-tree.h btrfs/transaction.h btrfs/volumes.h btrfs/ctree.h btrfs/extent_io.h btrfs/kerncompat.h btrfs/radix-tree.h btrfs/utils.h btrfs/disk-io.h btrfs/hash.h btrfs/list.h btrfs/rbtree.h btrfs/version.h
 
+AM_LDFLAGS = 
+AM_CFLAGS = 
+
 if ENABLE_STATIC
-LDFLAGS+=-static 
+AM_LDFLAGS+=-static 
 endif
 
 if ENABLE_TINFO
@@ -14,11 +17,11 @@ LIBS+=-ltinfo
 endif
 
 if ENABLE_MEMTRACE
-CFLAGS+=-DMEMTRACE -DMEMWATCH -DMW_STDIO
+AM_CFLAGS+=-DMEMTRACE -DMEMWATCH -DMW_STDIO
 endif
 
 if ENABLE_NCURSESW
-CFLAGS+=-I/usr/include/ncursesw
+AM_CFLAGS+=-I/usr/include/ncursesw
 endif
 
 version.h: FORCE
